File: src/main/java/com/application/vendetta/entities/Markers.java
package com.application.vendetta.entities;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

import java.util.List;


@Getter
@Setter
@Entity
@Table(name = "markers")
public class Markers {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;
    private String emailOfCreator;
    private String creationDate;
    private String description;
    @Column(unique=true)
    private String title;
    @Column(unique=true)
    private double lat;
    @Column(unique=true)
    private double lng;
    @ManyToOne
    private TypeOfMarker typeOfMarker;


}

File: src/frontend/src/app/models/markers.ts
import { TypeOfMarker } from './TypeOfMarker';
export class Markers {
    constructor
    (
     private emailOfCreator: string,
     private description: string,
     private title: string,
     private lat: number,
     private lng: number,
     private typeOfMarker: TypeOfMarker,
     private id?: number, private creationDate?: string,
    ) {
        this.emailOfCreator = emailOfCreator;
        
        this.title =  title
        this.lat = lat
        this.lng = lng
        this.typeOfMarker = typeOfMarker
        this.id = id
        let date = new Date()
        this.creationDate = (date).toISOString()
    }

}